src/main/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceImpl.java

```diff
@@ -292,20 +296,67 @@ public HearingResponse deleteHearingRequest(Long hearingId, DeleteHearingRequest
         validateVersionNumber(hearingId, deleteRequest.getVersionNumber());
         validateDeleteHearingStatus(hearingId);
         updateCancellationReasons(hearingId, deleteRequest.getCancellationReasonCode());
-        HearingEntity savedEntity = updateHearingStatus(hearingId);
+        HearingEntity savedEntity = updateHearingStatusAndVersionNumber(
```

Comment:
updateHearingStatusAndVersionNumber(Long hearingId, String newStatus) can be called as no need to update the request

Reply:
Agreed